dependency.lic

```diff
@@ -7,8 +7,9 @@ require 'net/http'
 require 'base64'
 require 'yaml'
 require 'ostruct'
+require 'digest/sha1'
 
-$DEPENDENCY_VERSION = '1.0'
+$DEPENDENCY_VERSION = '1.1'
```

Comment:
I have a dependency change incoming for bankbot to write to disk and was planning on using 1.1 for that. Just keep an eye out for whichever of these changes lands on master first.